routes/quizzes.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use App\Http\Controllers\QuizController;
+
+Route::multilingual('/quizzes/{quiz}', [QuizController::class, 'show'])
+    ->middleware('auth')
+    ->name('quizzes.show');
+
+Route::multilingual('/quizzes/{quiz}/result', [QuizController::class, 'storeQuizResult'])
+    ->method('post')
+    ->middleware('auth')
+    ->name('quizzes.show-result');
```

Comment:
I think the name here must be wrong as this appears to store the result, but the name is `show`.

Reply:
Another thing is you should follow the pattern in other route files to use groups.